sample-root-scripts/read_PMT.C

```diff
@@ -49,14 +49,13 @@ void read_PMT(char *filename=NULL) {
   // As you can see, there are lots of ways to get the number of hits.
   cout << "Number of tube hits " << wcsimrootevent->GetNumTubesHit() << endl;
   cout << "Number of Cherenkov tube hits " << wcsimrootevent->GetNcherenkovhits() << endl;
-  cout << "Number of tube hits " << wcsimrootevent->GetCherenkovHits()->GetEntries() << endl;
+  cout << "Number of Cherenkov tube hits " << wcsimrootevent->GetCherenkovHits()->GetEntries() << endl;
```

Comment:
I agree (but would like to agree on / implement the changes you suggested in #175 first, before adding better explanations here).